receiver/netflowreceiver/go.mod

```diff
@@ -0,0 +1,62 @@
+module github.com/open-telemetry/opentelemetry-collector-contrib/receiver/netflowreceiver
+
+go 1.22.0
+
+toolchain go1.22.2
+
+require (
+	github.com/stretchr/testify v1.9.0
+	go.opentelemetry.io/collector/component v0.107.1-0.20240816132030-9fd84668bb02
```

Comment:
Can we update these to the latest version (v0.113.0)? Some of modules will instead be v1.19.0, it may be worth it just to clear the dependencies and run a `go mod tidy` in this directory to pull in the latest.